moveos/moveos-stdlib/moveos-stdlib/sources/object.move

```diff
@@ -70,6 +78,10 @@ module moveos_std::object {
         self.owner = recipient;
     }
 
+    public fun public_transfer<T: key + store>(self: &mut Object<T>, recipient: address) {
```

Comment:
Make the `transfer` to `key + store` and remove `#[private_generics(T)]`, `public_transfer` to `transfer_extend` ? Keep consistency with `coin` and `account_coin_store`

Reply:
Does `xxx_extend` always have #[private_generics(T)] check? Is it more appropriate to put the object `public_transfer` function in the transfer module?

Reply:
The transfer function in the transfer module needs to use object_id as the argument.

```move
public entry transfer_object<T:key+store>(ctx, object_id, to){
  let object = context::borrow_object_mut<T>(ctx, object_id);
  object::transfer(object, to);
}
```

So, it depends on how to support the public borrow object from context.